data/json/construction.json

```diff
@@ -1951,7 +1951,7 @@
     "time": "60 m",
     "qualities": [ [ { "id": "HAMMER", "level": 1 } ], [ { "id": "DIG", "level": 2 } ] ],
     "components": [ [ [ "rock", 20 ], [ "2x4", 2 ] ] ],
-    "pre_flags": "DIGGABLE",
+    "pre_flags": { "flag": "DIGGABLE", "force_ter": true },
```

Comment:
Why not to use full word, i.e. `force_terrain`?

Reply:
Sorry, I'm used to using shorthand variables.

Reply:
```suggestion
    "pre_flags": { "flag": "DIGGABLE", "force_terrain": true },
```